tests/e2e-leg-11/client-proxy/45-assert.yaml

```diff
@@ -0,0 +1,67 @@
+# (c) Copyright [2021-2024] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: v-client-proxy-pri1
+status:
+  currentReplicas: 4
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: v-client-proxy-sec1
+status:
+  currentReplicas: 0
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: v-client-proxy-pri1-proxy
+status:
+  readyReplicas: 2
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: v-client-proxy-pri1-proxy-cm
+immutable: false
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: v-client-proxy-sec1-proxy
+status:
+  readyReplicas: 0
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: v-client-proxy-sec1-proxy-cm
+immutable: false
+---
+apiVersion: vertica.com/v1
+kind: VerticaDB
+metadata:
+  name: v-client-proxy
+spec:
+  subclusters:
+    - name: pri1
+      size: 4
+      proxy:
+        replica: 2
+    - name: sec1
+      size: 0
+      proxy:
+        replica: 0
```

Comment:
You need to assert the subcluster status as well.

Reply:
Updated in https://github.com/vertica/vertica-kubernetes/pull/980/commits/2ba5e9177241b32e2fa786d009b93a1fab9a0849